.github/workflows/ci.yml

```
@@ -170,7 +170,7 @@ jobs:
           # ${{ env.stepName }}
           echo -e "::group::${{ env.bashInfo }} ${{ env.stepName }} ${{ env.bashEnd }}"
 
-          cmake -S . -B build -DFREERTOS_PLUS_TCP_TEST_CONFIGURATION=ENABLE_ALL
+          cmake -S . -B build -DFREERTOS_PLUS_TCP_BUILD_BUILD_TEST=ON -DFREERTOS_PLUS_TCP_TEST_CONFIGURATION=ENABLE_ALL
```

```suggestion
          cmake -S . -B build -DFREERTOS_PLUS_TCP_BUILD_TEST=ON -DFREERTOS_PLUS_TCP_TEST_CONFIGURATION=ENABLE_ALL
```
The changes seems fine to me, one little remark can we make the above change instead of  DFREERTOS_PLUS_TCP_**BUILD_BUILD**_TEST

---

`FREERTOS_PLUS_TCP_BUILD_TEST` already exists and is used for a different purpose (unit test build enable). Would changing `FREERTOS_PLUS_TCP_BUILD_BUILD_TEST` to `FREERTOS_PLUS_TCP_ENABLE_BUILD_CHECKS` be acceptable? This option will be used for what `test/build-combination/README.md` refers to as "Build checks" and the CI job that will use this option is the `build-checks` CI job.